Experience.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title> Experience</title>
+        <title> experience</title>
        <link rel="stylesheet" href="reem.CSS">
-       
+       </head>
+       <body>
+          <main >
+        <header>
+         <h1>Experience</h1>
+        </header>
 
-    </head>
-    <main >
-       <h1>Experience</h1>
-       <br><br>
-       <h2 class="aldi">permanent store assistant (ALDI) </h2>
+       <nav>
+        <h2>permanent store assistant (ALDI) </h2>
```

Comment:
Can drop the space character ` ` after the `)` e.g. becomes `<h2>permanent store assistant (ALDI)</h2>`.